src/BlazorWebView/src/Microsoft.AspNetCore.Components.WebView.Maui/Windows/WebView2WebViewManager.cs

```diff
@@ -113,6 +113,13 @@ private async Task InitializeWebView2()
 
 			_webview.CoreWebView2.WebMessageReceived += (sender, eventArgs)
 				=> MessageReceived(new Uri(eventArgs.Source), eventArgs.TryGetWebMessageAsString());
+
+			_webview.CoreWebView2.DOMContentLoaded += async (_, __) =>
+			{
+				await _webview.CoreWebView2!.ExecuteScriptAsync(@"
+					window.Blazor.start();
```

Comment:
```suggestion
					Blazor.start();
```

... is equivalent, unless there's something subtle I'm missing!

Reply:
Steve, what you seem to be missing is that this is why Blazor exists, so that people don't have to know this difference 😁